articles/virtual-desktop/troubleshoot-vm-configuration.md

```diff
@@ -337,6 +337,10 @@ If your users see an error that says, “We couldn't connect to the remote PC be
 
 To learn more about this policy, see [Allow log on through Remote Desktop Services](/windows/security/threat-protection/security-policy-settings/allow-log-on-through-remote-desktop-services).
 
+## Issues with golden image
```

Comment:
```suggestion
## I can't deploy the base image
```

Reply:
Same as above - golden image and base image are different.